src/components/dialogs/field-hook.js

```diff
@@ -62,21 +50,13 @@ export const useTextValue = ({
                 style={{ width: '100%' }}
                 onChange={handleChangeValue}
                 FormHelperTextProps={{
-                    className: classes.helperText,
+                    sx: styles.helperText,
```

Comment:
`className` was not working, because the correct prop is `classes` : https://mui.com/material-ui/api/form-helper-text/
So you can remove the `sx`

Reply:
I removed the useless FormHelperTextProps.